hack/upgrade-cnv.sh

```diff
@@ -1,3 +1,105 @@
 #!/usr/bin/env bash
 
-echo "this is a placeholder until the upgrade test lane is implemented"
+set -euo pipefail
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+echo "OCP_VERSION: $OCP_VERSION"
+
+eval "$(jq -r '."'"$OCP_VERSION"'" | to_entries[] | .key+"="+.value' version-mapping.json)"
+
+# These variables are set when the eval above is executed
+# shellcheck disable=SC2154
+echo "Using index_image: $index_image"
+# shellcheck disable=SC2154
+echo "Using bundle_version: $bundle_version"
+
+#==========================================
+# Deploy HCO using latest released version
+#==========================================
+
+echo "creating $TARGET_NAMESPACE namespace"
+oc create ns $TARGET_NAMESPACE
```

Comment:
on a serious note: maybe we should just run the scripts using -x so we get rid of all the echo stuff and see what actually is being executed. WDYT?

Reply:
How about using both ehco's and -x? The echo's are still useful for someone who isn't proficient in the test scripts...